src/main/kotlin/org/lsst/dax/albuquery/QueryMetadataHelper.kt

```diff
@@ -87,6 +87,7 @@ class QueryMetadataHelper(val analyzer: Analyzer.TableAndColumnExtractor) {
             val columnMetadata =
                 ColumnMetadata(name,
                     datatype = metaservColumn?.datatype ?: jdbcToLsstType(jdbcColumn.jdbcType),
+                    description = metaservColumn?.description,
```

Comment:
I think you want this to be:

`metaservColumn?.description ?: ""`

which makes sure it's never null (As per SUIT)

Reply:
Good point - will make this change.